modules/installation-creating-azure-control-plane.adoc

```diff
@@ -64,10 +64,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/05_masters.json" \
   --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2>
+  --parameters masterVMsize <3>
```

Comment:
```suggestion
  --parameters masterVMSize <3>
```

Reply:
@r4f4 Is there a value I can add to this one as well?